app/src/main/java/com/example/mytrainingpal/util/Utils.kt

```diff
@@ -1,13 +1,35 @@
 package com.example.mytrainingpal.util
 
+import java.time.LocalDate
 import java.time.LocalDateTime
 import java.util.*
 
-fun todayDate(): Date {
-    val todayDate = LocalDateTime.now()
+fun localDateToJavaDate(date: LocalDate): Date{
     return GregorianCalendar(
-        todayDate.year,
-        todayDate.monthValue,
-        todayDate.dayOfMonth
+        date.year,
+        date.monthValue - 1,
+        date.dayOfMonth
     ).time
 }
```

Comment:
A pretty important bugfix. The added muscle pain entries were always shifted one month ahead 😅